e2e/tests/functional/plugins/imagery/exampleImagery.e2e.spec.js

```diff
@@ -28,11 +28,11 @@ but only assume that example imagery is present.
 import { createDomainObjectWithDefaults, setRealTimeMode } from '../../../../appActions.js';
 import { waitForAnimations } from '../../../../baseFixtures.js';
 import { expect, test } from '../../../../pluginFixtures.js';
-const backgroundImageSelector = '.c-imagery__main-image__background-image';
 const panHotkey = process.platform === 'linux' ? ['Shift', 'Alt'] : ['Alt'];
 const tagHotkey = ['Shift', 'Alt'];
 const expectedAltText = process.platform === 'linux' ? 'Shift+Alt drag to pan' : 'Alt drag to pan';
 const thumbnailUrlParamsRegexp = /\?w=100&h=100/;
+const EXTENDED_TIMEOUT = 30 * 1000;
```

Comment:
let's make this more descriptive of what this timeout is actually used for or add a comment in the line where this is used